documentation/Hierarchical File System (HFS).asciidoc

```diff
@@ -67,16 +67,16 @@ Email change
 == Overview
 
 The Hierarchical File System (HFS) is mainly used on the Macintosh platform.
-The Mactintosh File System (MFS) is the predecessor of HFS.
+The Macintosh File System (MFS) is the predecessor of HFS.
 
 [cols="1,5",options="header"]
 |===
 | Version | Introduced in
 | MFS | 400 KiB floppies
 | HFS | Early Mac OS
 | HFS+ 8.10 | Mac OS 8.1 to 9.2.2
-| HFS+ 10.0 | Mac OS 10.0
-| HFSX | Mac OS 10.3
+| HFS+ 10.0 | Mac OS X 10.0
```

Comment:
Mac OS is used as the generic name for the operating system family. I'll add a note.